<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>777</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1147</width>
    <height>777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChatMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>841</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>680</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
      
           color: white;
           border: none;
           border-radius: 5px;
           padding: 10px;
           cursor: pointer;
           margin-top: 20px;
           text-align: center;
          font-family: 'Microsoft YaHei Light';color: #000;

margin: 0;
    padding: 0;
    width: 250px;
    height: 40px;
    outline: none;
    border: 1px solid #ccc;
    border-radius: 20px;
    letter-spacing: 5px;
    color: #333;
    background: none;
    cursor: pointer;
    margin-top: 20px;</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="messageTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>821</width>
       <height>161</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>499</y>
       <width>831</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QToolButton" name="faceButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-2</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>表情</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/face.png</normaloff>:/images/face.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="fileButton">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>-2</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>发送文件</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>34</width>
        <height>34</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>9</y>
       <width>821</width>
       <height>491</height>
      </rect>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>21</y>
         <width>821</width>
         <height>461</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
    <zorder>messageTextEdit</zorder>
    <zorder>sendButton</zorder>
    <zorder>widget_3</zorder>
    <zorder>stackedWidget</zorder>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>49</y>
      <width>311</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>1</y>
       <width>291</width>
       <height>671</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeWidget {
    
	
	background-color: rgb(243, 243, 243);
    color: var(--UI-FG-0); /* 90%不透明的黑色文本 */
    font-size: 28px; /* 设置字体大小为28px */
    font-family: &quot;Microsoft YaHei Light&quot;, Helvetica, sans-serif; /* 字体 */
}

QTreeWidget::item {
    padding: 10px;
    border: none;
}

QTreeWidget::item:selected {
    background-color: var(--UI-BG-4); /* 深灰色背景 */
    color: var(--UI-FG-0); /* 90%不透明的黑色文本 */
}

QTreeWidget::item:hover {
    background-color: var(--UI-BG-3); /* 浅灰色背景 */
    color: var(--UI-FG); /* 黑色文本 */
}
</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
